dataprofiler/data_readers/csv_data.py

```diff
@@ -58,8 +65,9 @@ def __init__(self, input_file_path=None, data=None, options=None):
         :return: None
         """
         options = self._check_and_return_options(options)
+        options = cast(Dict, options)
```

Comment:
shouldn't the line above already indicate that it is a dict?

Reply:
You're right. I probably forgot to delete this from a previous implementation.